src/binder/bind/bind_export_database.cpp

```diff
@@ -30,6 +30,9 @@ static std::vector<ExportedTableData> getExportInfo(const Catalog& catalog, Tran
     Binder* binder) {
     std::vector<ExportedTableData> exportData;
     for (auto tableEntry : catalog.getTableEntries(tx)) {
+        if (catalog.tableInRDFGraph(tx, tableEntry->getTableID())) {
```

Comment:
You don't have to add this if statement.
binder->bindExportTableData
will return false if we are not going to export the table:
1. if the table has only one serial column.
2. RDF table